### ISSUE TITLE
500 now returned on duplicates

### ISSUE THREAD

**Issue by k1o0** [cortex-lab/alyx#548]
On Alyx dev I now get a 500 response when re-registering a file.  I'm unsure if this a bug or a feature! Previously decided Alyx would simply return the pre-exisiting file records. This has been useful for bulk registration of ALF files (basically patching up the sessions that have missing file records).  The code could be updated to check Alyx and drop any file names from the request which have already been registered.  What are your opinions on this?

```
IntegrityError at /register-file
     duplicate key value violates unique constraint "data_filerecord_data_repository_id_relative_path_ecbcbe70_uniq"
     DETAIL:  Key (data_repository_id, relative_path)=(b2e1630f-458d-48bf-be8f-2c42d9b7afd6, PC043/2019-05-11/5/2019-05-11_5_PC043_parameters.json) already exists.
```

**Comment by oliche** [comment]
Definitively a bug

**Comment by k1o0** [comment]
Besides `2019-05-11_5_PC043_parameters.json` I also tried one for LEW011:
`{"created_by":"miles","hostname":"zubjects.cortexlab.net","path":"LEW011\\2018-11-15\\2","filenames":["2018-11-15_2_LEW011_parameters.json"]}`

**Comment by oliche** [comment]
@k1o0 it's because the original user was not the same of the actual (Morgane versus miles).
It should not throw an error anyway.

I've pushed something on dev cortexlab, let me know if it works OK (it was fine with my tests).

**Comment by k1o0** [comment]
Yep, it appears to be working now. @nsteinme has this fixed your registration problem also?

**Comment by rossant** [comment]
I pushed something on alyx-dev cortexlab, could you try it?
